client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -152,7 +153,14 @@ public RestService(String baseUrlConfig) {
         return null;
       } else {
         InputStream es = connection.getErrorStream();
-        ErrorMessage errorMessage = jsonDeserializer.readValue(es, ErrorMessage.class);
+        ErrorMessage errorMessage;
+        try{
+          errorMessage = jsonDeserializer.readValue(es, ErrorMessage.class);
+        }
+        catch (JsonProcessingException e) {
+          int unable_to_parse_json_errror_code = 50005;
```

Comment:
Nit: make this a `private static final` class member since its a constant? Also, I assume this just didn't make it into the `Errors` class since it is on the client side and not in the `core` project? If we're going to add this code, we might want to add a note there to make sure we don't accidentally reuse the same error code.